command-builder/deploy/schema.d.ts

```diff
@@ -0,0 +1,7 @@
+export interface Schema {
```

Comment:
Minko used `json2ts` for that. Before each build this file is then auto-generated, so that's why it's not required to be under source control.

Reply:
I will do this in the next PR.